.maestro/tests/AuthenticationRejectedWithChallenge.yaml

```diff
@@ -34,32 +34,4 @@ appId: com.basistheory.threeds.example
 - hideKeyboard
 - assertVisible: "rejected"
 - assertVisible: "suspected-fraud"
-- tapOn: "Clear"
-
-# Mastercard
-
-- tapOn: "Token ID"
-- inputText: "fe819a3e-bd63-4297-bef2-dc3a41e8ae37" #  4444333322221111 Visa
```

Comment:
something is wrong w/ this token and this card - the card won't even tokenize for me - we don't need 2 tests for this scenario anyway so removed it